share/spice/piratetalk/piratetalk.js

```diff
@@ -0,0 +1,28 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_piratetalk = function(api_result) {
+
+        if (!api_result || api_result.translation==="") {
+            return Spice.failed('piratetalk');
+        }
+        
+        Spice.add({
+            id: "piratetalk",
+            name: "Answer",
+            data: {
+                title:api_result.translation,
+            },
+            meta: {
+                sourceName: "Pirate",
+                sourceUrl: 'https://pirate-speak.herokuapp.com/',
+            },
+            templates: {     
+                group: 'base',
```

Comment:
@thejasbabu In that case we need to modify the CSS to get rid of any quote-marks and 'quote-like' items - we just want the text.